### PR TITLE
Capitalized first letter in each line of the list

### LINKED ISSUE #35455: Capitalize list items in committing-changes-to-a-pull-request-branch-created-from-a-fork.md
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork

This issue is intended for newcomers to the OS Docs repo! 💛

If you are veteran of the OS Docs repo and are looking for issues to work on, take a look at our [help wanted section](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) ✨

### What part(s) of the article would you like to see updated?

> You can only make commits on pull request branches that:
> 
> - are opened in a repository that you have push access to and that were created from a fork of that repository
> - are on a user-owned fork
> - have permission granted from the pull request creator
> - don't have [branch restrictions](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#restrict-who-can-push-to-matching-branches) that will prevent you from committing

Capitalize the first letter in each line of this list to maintain consistency with other docs (and [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#lists))

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by VictorDoyle**
### Why:

Closes: https://github.com/github/docs/issues/35455

### What's being changed:

Capitalized the first letter in each line of the list in doc page https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork  to maintain consistency with other docs. Screenshot below showcases the changes made: 
Screenshot showing capitalized letters in list items

### Check off the following:

- [x] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require a SME review from GitHub staff.
- [x] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [x] All CI checks are passing.

### CODE REVIEW

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ topics:
 shortTitle: Commit to PR branch from fork
 ---
 You can only make commits on pull request branches that:
-* are opened in a repository that you have push access to and that were created from a fork of that repository
-* are on a user-owned fork
-* have permission granted from the pull request creator
-* don't have [branch restrictions](/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#restrict-who-can-push-to-matching-branches) that will prevent you from committing
```
[the remark this answers is not in the document]
Comment (alfonssoo-sys) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md:21]: ...

Comment (alfonssoo-sys) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md:21]: ...

Comment (alfonssoo-sys) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md:21]: Nice

Comment (VictorDoyle) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md:21]: Resolving as spam

[the remark this answers is not in the document]
Comment (VictorDoyle) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork.md:21]: Resolving as spam

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@VictorDoyle Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Pull request merged**